apps/VideoStream/videoStream.py

```diff
@@ -68,10 +69,16 @@ def send_frame(self, frame, address, quality=4):
         frame = zlib.compress(frame, -1)
         self.socket.sendto(frame, address)
 
+    def listen_thread(self):
+        thread = threading.Thread(target = self.listen)
+        thread.daemon = False
+        thread.start()
+
     def listen(self, port=None):
         if port is None:
             port = self.port
-        self.socket.bind(('', port))
+        with threading.Lock():
```

Comment:
Might be nice to put a short sleep cycle in this loop, maybe a second or 2 so that it does not take up too many resources on the pi.